pkg/driver/driver.go

```diff
@@ -886,6 +886,15 @@ func (d Driver) CreateSnapshot(ctx context.Context, req *csi.CreateSnapshotReque
 		return nil, missingAttr("CreateSnapshot", req.GetName(), "Name")
 	}
 
+	d.log.WithField("req.parameters", req.GetParameters()).WithField("req.secrets", req.GetSecrets()).Debug("parsing request")
+
+	params, err := volume.NewSnapshotParameter(req.GetParameters(), req.GetSecrets())
+	if err != nil {
+		return nil, status.Errorf(codes.InvalidArgument, "invalid snapshot parameter: %v", err)
```

Comment:
```suggestion
		return nil, status.Errorf(codes.InvalidArgument, "invalid snapshot parameter: %w", err)
```
Supported here?

Reply:
Seems to use `fmt.Sprintf` internally, so no, i.e. it would just be an `%v`